src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -84,6 +84,7 @@ public SentryUnityOptions()
         {
             // IL2CPP doesn't support Process.GetCurrentProcess().StartupTime
             DetectStartupTime = StartupTimeDetectionMode.Fast;
+            CacheDirectoryPath = Application.persistentDataPath;
```

Comment:
I wonder if we might have multiple instances of the same game open and that would conflict here.
It isnt' a problem on mobile but it could happen on desktop. @Tyrrrz would the .NET SDK handle that today?

Reply:
No, it won't. We decided not to add it for now.

Reply:
Yeah. That makes me wonder if we need to solve this now, or just document this.

> Note: For Desktop games: If more than one instance of your game can run at the same time, you'll need to pass a separate path explicitly.

Or something along those li es

Reply:
It's not a terribly hard problem to solve i think

Reply:
We talked briefly and it was a bit annoying. Since crashes are sent after restart, we need a segment of the path that survives restart. So `pid` won't do for example. Not a hard problem but not trivial as 'add a hash of X in there' kind of trivial